ethminer/MinerAux.h

```diff
@@ -893,11 +894,9 @@ class MinerCLI
 			results.push_back(rate);
 			mean += rate;
 		}
-		results.sort();
+		sort(results.begin(), results.end());
 		cout << "min/mean/max: " << results.front() << "/" << (mean / _trials) << "/" << results.back() << " H/s" << endl;
-		results.pop_front();
-		results.pop_back();
-		for (auto it = results.begin(); it != results.end(); it++)
+		for (auto it = results.begin()+1; it != results.end()-1; it++)
```

Comment:
This will not work if there is a single result. Why do you skip the first and last items?

Reply:
Because the point of the inner mean is to discard the highest and lowest value, as to get a sample less influenced by outliers. This was also the case in the old implementation.

Of course, you are right that this causes unacceptable errors when run with 1 or 2 trials. In these cases, however, the inner mean is undefined. What do you propose should happen then?

- Don't print the inner mean at all
- Print 0 as the inner mean
- Print the regular mean as the inner mean
- Print a message stating the undefinedness of the inner mean

Reply:
Done